episodes/collaborating-team.md

```diff
@@ -0,0 +1,256 @@
+to make changes to a lesson directly on the `main` branch of the repository,
+doing so is not recommended:
+
+1. It increases the likelihood of unintentional conflicts being created when two
+   collaborators work on the same file simultaneously.
+1. It does not provide an opportunity for changes to be checked and discussed
+   before they are included in the lesson.
+   This makes it much more likely that mistakes will be included in the lesson,
+   and that the commit history of the project will include many
+   changes made only to fix/polish/undo previous commits.
+1. It prevents your team from benefiting from the powerful features that Git and GitHub
+   provide to support and foster collaboration.
+
+Instead, it is recommended practice to do most work in _branches_:
+sets of changes that build on the work in the `main` branch (or another)
+but are not included in that branch until they are explicitly _merged_ into it.
+GitHub provides a way for this merging process to be managed,
+by giving a platform for the changes to be described, discussed and reviewed,
+before a decision is made to integrate or not integrate the changes
+in the default branch of the project and the associated lesson.
+On GitHub, these units of discussion, review,
+and acceptance/rejection of the changes within a branch
+are called _Pull Requests_.
+
+:::::: callout
+
+## Why "Pull Request"?
+
+The term _Pull Request_ originates from the early days of GitHub, when
+a contributor would be requesting that the project maintainer(s)
+use Git to fetch (download) their branch and merge the changes
+into the main branch of the project.
+In the terminology of Git, this combination of fetching and merging a set of changes
+is called _pulling_.
+The name Pull Request is slightly anachronistic now, because a project maintainer can often
+manage the whole process of reviewing a contribution in GitHub's web interface.
+
+Note: A common nickname for a _Pull Request_ is PR or PRs for _Pull Requests_.
+
+::::::::::::::
+
+
+### Protecting `main` Branch
+
+:::::::::::::::::: instructor
+
+When preparing to teach this section, you should check that you have admin privileges on
+your example repository.
+
+:::::::::::::::::::::::::::::
+
+As the maintainer of a project, you can enforce this workflow by
+_protecting_ the default branch of the GitHub repository:
+this will prevent anyone without _Admin_ access from committing changes directly to
+that branch.
+
+To activate this branch protection:
+
+1. Open the _Settings_ tab of the repository and select _Branches_ in the left sidebar,
+under the _Code and automation_ section.
+2. Click the "Add branch protection rule" button.
+3. Type the name of the branch your GitHub Pages are being built from (`main`)
+   into the "Branch name pattern" box.
+4. Check the "Require a pull request before merging" option,
+   but leave the others unchecked for now because we don't need them for this project.
```

Comment:
This can wait as a separate PR but in #193 we discussed making this a call out instead of a demo and that could also be added to this PR if we wanted to

Reply:
This is now a separate PR  https://github.com/carpentries/lesson-development-training/pull/259 that will get merged on top of this one.